pkg/controllers/dnshealthcheckprobe/suite_test.go

```diff
@@ -0,0 +1,142 @@
+// go:build integrationhealth
```

Comment:
Did you intend to add a different tag, or should it be `//go:build integration`?